de.json

```diff
@@ -151,9 +151,9 @@
         "World.Actions.HostingWarning" : "Sie sind Host dieser Welt. Alle Nutzer werden getrennt, wenn Sie die Welt schließen.",
 
         "World.Actions.GetWorldOrb" : "Welt-Orb erzeugen",
-        "World.Actions.CopyWorldURL" : "World-URL in Zwischenablage kopieren",
+        "World.Actions.CopyWorldURL" : "Welt-URL in Zwischenablage kopieren",
         "World.Actions.CopyRecordURL" : "Aufzeichnungs-URL in Zwischenablage kopieren",
-        "World.Actions.StartCustomSession" : "Starte nutzerdefinierte Sitzung",
+        "World.Actions.StartCustomSession" : "Starte benutzerdefinierte Sitzung",
```

Comment:
I used "Benutzerdefinierte Sitzung starten" for this one, both of them are essentially the same.